### ISSUE TITLE
Problems with list UI: Ctrl-F (in browser) not working as expected

### ISSUE THREAD

**Issue by chepec** [plainmade/unmark#206]
When the list of links is long, the way it is loaded(dynamically, somehow) does not work well with the browser's in-page search function (tested both Chrome and Firefox). In short, if the search text is not yet visible on the screen, the in-browser search function finds no matches (although the text exists in the title of a link item further down the list). This is not the expected behaviour. 

Yes, I'm aware unmark has its own search function, but why break the in-browser search? Could we unbreak it please?

**Comment by cdevroe** [comment]
@chepec Do you mean that due to pagination the in-browser search function does not work? Or, that even if the title of the mark is shown in the list that Chrome cannot find the list item?

**Comment by chepec** [comment]
Yes, pagination. Using the in-browser search works fine as long the title is visible on the screen or has been "scrolled past" during the current session (as far as I can tell).

And this is what disturbs me from a user experience point of view. It causes the in-browser search to work very inconsistently (effectively depending on how far I've happened to scroll) and this leads to all kinds of confusion ;-) I really appreciate you taking the time to consider this issue, whatever you decide to do about it.
